### PR TITLE
Feature/create base module structure gradle

### LINKED ISSUE #2: Create base modules structure with gradle as build tool
We agree to work with following project structure:

* `rush` (main project catalog)
  * `model` (_will contains proto files with model definition_)
  * `services` (_will contains proto files with services definition_)
  * `shopping` (_will contains shopping domain implementation_)
  * `products` (_will contains product domain implementation_)
  * `ordering` (_will contains ordering domain implementation_)
  * `zero-waste` (_will contains zero-waste domain implementation_)
  * `accounts` (_will contains accounts domain implementation_)
  * `qa` (_will contains functional tests to verify that services are working as a whole_)

Rules:
* main project should be organized as multi module gradle project,
* each underlying submodule is separate gradle project,
* `model` submodule should has no dependencies to other submodules,
* `services` submodule should has dependency to `model` submodule,
* all domain releated submodules should has dependency to both `model` and `services` submodules,
* `qa` should has dependency to both `model` and `services` submodules.

### LINKED ISSUE #3: Add pipeline to build project
We agree to configure single pipeline based on github actions to build whole project. As a whole project we understand main project module `rush` and all its submodules (like `model`, `services` etc.).

Pipeline should be configured for `main` branch on `push` event.

### PR DESCRIPTION
**Pull request by pgancarz**
fixes #2, fixes #3

### CODE REVIEW

**Review by wisadr — APPROVED** [review]
cacy

**Comment by pgancarz** [comment]
@honestit MIchał, somethingis wrong, I have to approvals and still cannot merge this PR.

**Review by wisadr — APPROVED** [review]
cacy#2

**Pull request merged**
